scss/jetpack-connect.scss

```diff
@@ -0,0 +1,28 @@
+// ==========================================================================
+// TOS Blurb
+// ==========================================================================
+
+.jp-jetpack-connect__tos-blurb {
+  font-size: .75rem;
```

Comment:
Does it make sense to make the styles the same as the `jp-banner__tos-blurb`? Namely:

```
.jp-banner__tos-blurb {
	display: block;
	padding: rem( 6px ) 0;
	line-height: 1.5;
	font-size: rem( 11px );
	color: $gray-dark;
	@include minbreakpoint(phablet) {
		margin-left: rem(18px);
	}

	a {
		color: inherit;
	}
}
```

Reply:
I don't think so in this case. Basically all of those styles are already inherited by `.jp-jetpack-connect__tos-blurb` and we don't want to have a left margin on any breakpoint.

The only difference is the `font-size` which is `0.75rem => 12px`. The size of 12 pixels is taken from the Figma mock up.

I think that as long as it looks and works as expected it doesn't make sense add up new CSS rules, but I'm happy to adjust.

Reply:
Happy to keep it as-is then 👍